src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -2,11 +2,8 @@
 {%- import "static/webpack-macros.html" as _webpack with context %}
 
 {%- block css %}
-    {{ _webpack.head_pre_bootstrap() }}
+    {{ _webpack.head_pre_assets() }}
     {{ _webpack.head_pre_icons() }}
-    {% block fonts %}
```

Comment:
Can you explain more about the reasons for this fonts block removal?

Reply:
This theme used to include its own custom font files, which I think is why this block was here. However, this was removed in https://github.com/pydata/pydata-sphinx-theme/pull/285 so I think that this template stuff is cruft that isn't used anymore. I figured we should remove things in general that we're not using.

Reply:
Thanks for the additional information!